libs/moment-adapter/src/lib/moment-datetime/moment-datetime-adapter.ts

```diff
@@ -0,0 +1,347 @@
+import { Inject, Injectable, InjectionToken, LOCALE_ID, Optional } from '@angular/core';
+import moment, { Locale, LongDateFormatSpec, Moment, MomentFormatSpecification, MomentInput } from 'moment';
+import { DatetimeAdapter, DateLocale, MonthLocaleType } from '@fundamental-ngx/core/datetime';
+
+function range<T>(length: number, mapFn: (index: number) => T): T[] {
+    return Array.from(new Array(length)).map((_, index) => mapFn(index));
+}
+
+export interface MomentDatetimeAdapterOptions {
+    strict?: boolean;
+    useUtc?: boolean;
+}
+
+export const MOMENT_DATE_TIME_ADAPTER_OPTIONS = new InjectionToken<MomentDatetimeAdapterOptions>(
+    'MOMENT_DATE_TIME_ADAPTER_OPTIONS', {
+        providedIn: 'root',
+        factory: MOMENT_DATE_TIME_ADAPTER_OPTIONS_FACTORY
+    }
+);
+
+export function MOMENT_DATE_TIME_ADAPTER_OPTIONS_FACTORY(): MomentDatetimeAdapterOptions {
+    return {
+        useUtc: false,
+        strict: false
+    };
+}
+
+@Injectable()
+export class MomentDatetimeAdapter extends DatetimeAdapter<Moment> {
+
+    private _momentLocaleData: Locale;
+    private _localeData: DateLocale;
+    public fromNow: undefined;
```

Comment:
what is this for?

Reply:
I guess for this:
![image](https://user-images.githubusercontent.com/5276786/126723591-c83a9ee6-954b-4203-9aa7-70fe0d1863ae.png)

Reply:
@dimamarksman I don't know anything about this feature, fd also don't have implementation of this abstraction.
![image](https://user-images.githubusercontent.com/5276786/126723717-20863fa1-5deb-46dc-a7a5-9d82f2a6e5de.png)

Reply:
I just saw that it was introduced [here](https://github.com/SAP/fundamental-ngx/pull/5502):
>  Optional fromNow function was added to the DatetimeAdapter abstract class. Because there are no truly "optional" abstract functions, any adapter that implements the DatetimeAdapter will need to implement the fromNow function - however if the developer does not wish to use the fromNow function, in their custom adapter they can simply implement it as fromNow: undefined as I've done here for the default FdDatetimeAdapter

Reply:
@mikerodonnell89 should it be implemented in scope of this ticket?

Or in the scope of this: https://github.com/SAP/fundamental-ngx/pull/5502 + documentation?

Reply:
since dateFromNow pipe is not documented yet let's keep it undefined here as well and create a separate issue for that